app/policies/auth_verification_policy.rb

```diff
@@ -0,0 +1,12 @@
+class AuthVerificationPolicy
+  def self.verified?(auth)
```

Comment:
It feels like this method will be really huge after adding a little bit more oauths
WDYT about this realization?

```ruby
def self.verified?(auth)
  request_verification_for(auth) || fail_with_error(auth)
end

private

def request_verification_for(auth)
  send(auth.provider)
end

def fail_with_error(auth)
  fail ArgumentError, I18n.t("omniauth.verification.not_implemented", kind: auth.provider)
end

def facebook
...
end
```